packages/kolibri-common/components/SyncSchedule/EditDeviceSyncSchedule.vue

```diff
@@ -0,0 +1,395 @@
+<template>
+
+  <ImmersivePage
+    :appBarTitle="$tr('toolbarHeader')"
+    :route="backRoute"
+    :icon="icon"
+  >
+    <KPageContainer v-if="device" :style="pageHeight">
+      <KGrid gutter="48" class="edit-sync-schedule">
+
+        <KGridItem>
+          <h1>{{ $tr('editSyncScheduleSubTitle') }}</h1>
+        </KGridItem>
+
+        <KGridItem>
+          <p>{{ device.device_name }} </p>
+        </KGridItem>
+
+        <KGridItem class="select-element">
+          <div
+            class="k-select"
+          >
+            <KSelect
+              v-model="selectedInterval"
+              class="selector"
+              :style="selectorStyle"
+              :options="selectArray"
+              label="Repeat"
+            />
+
+          </div>
+          <div
+            v-if="selectedItem.label !== 'Never'
+              && selectedItem.label !== 'Every hour'
+              && selectedItem.label !== 'Every day'"
+            class="k-select next-k-select-1"
+          >
+            <KSelect
+              v-model="selectedDay"
+              class="selector"
+              :style="selectorStyle"
+              :options="getDays"
+              label="On"
+            />
+
+          </div>
+          <div
+            v-if="selectedItem.label !== 'Never' && selectedItem.label !== 'Every hour' "
+            class="k-select next-k-select-2"
+          >
+            <KSelect
+              v-model="selectedTime"
+              class="selector"
+              :style="selectorStyle"
+              :options="SyncTime"
+              label="At"
+            />
+          </div>
+
+        </KGridItem>
+
+        <KGridItem class="more-spacing">
+
+          <p class="spacing">
+            {{ $tr('serverTime') }}
+            {{ now }}
+          </p>
+
+          <p class="spacing">
+            <KCheckbox>
+              {{ $tr('checboxlabel') }}
+            </KCheckbox>
+          </p>
+          <KButton
+            :vIf="msg"
+            appearance="basic-link"
+            class="spacing"
+            @click="removeDevice"
```

Comment:
No need to change, but you can also do direct assignment in `@click` handlers, so instead of creating an intermediary method, you could do:
```
@click="removeDeviceModal = true"
```

and it would work just the same!